src/createServer.js

```diff
@@ -1,3 +1,68 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const targetCase = url.searchParams.get('toCase');
+    const errors = [];
+
+    res.setHeader('Content-Type', 'application/json');
+
+    if (!originalText) {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!targetCase) {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (
+      !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(targetCase)
+    ) {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+      });
+    }
```

Comment:
It would be better to extract this validation logic into a separate function. This would make the code easier to read and test, and would also allow you to reuse the validation logic elsewhere if needed in the future.